### ISSUE TITLE
mAP 欧尼

### ISSUE THREAD

**Issue by moothes** [princeton-vl/CornerNet#58]

**Comment by moothes** [comment]
Moreover, I use the FPN as backbone and achieve 63.25 mAP on 07test set.

**Comment by xingyizhou** [comment]
I tried fine-tuning the provided CornerNet_500000 on VOC with two GPU for one day, it gives around 81 mAP. Training from scratch on 2 GPUs for one week (250k iters) on VOC gives about 70 mAP according to my preliminary experiments.

**Comment by oujieww** [comment]
I think the reason should be pretrain-weights? for ssd,fssd,rfbssd, they used imagenet pretrained vgg as init weights,but for hourglass-model, there are not pretrained weights.coco dataset is big enough,but voc dataset is not as big as coco

**Comment by Xudangliatiger** [comment]
。。so the paper did not try voc07, guess that voc is not big enough to train from scratch

**Comment by TEOEFW** [comment]
How can i train cornetNet in on VOC data?

**Comment by Complicateddd** [comment]
> I train the source code on 07train+12train sets of PASCAL VOC, and test on 07test set. The mAP is **66.13**. Training loss is around **1** while test loss between **2-5**. The batch size is set to 11 due to the limitation of our GPU. Other settings are the same as in COCO. I miss something? Anyone has better results?
can u tell me how to modify the code to test voc dataset on a train model in voc

**Comment by Glutton-zh** [comment]
i train on voc2007 too but i just get a rusult.json but map, how can i get the map about my model
